api/PaymentRequest.json

```diff
@@ -866,6 +1068,378 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "detailsPromise_displayItems_parameter": {
```

Comment:
This structure is the same as we'd have for `request.show({ displayItems: ...})` but the parameter here is a promise, not an object (IDL dictionary). So... I'm not sure if this needs an update to the guideline or  something.

Before going down that path, is it really necessary to represent this parameter at all? The data looks almost the same as the parent feature, if it's only a small difference between Chrome 60 and 61 I don't think that warrants a subfeature, and not even a note on the main feature.